sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/models/KeyVaultRoleAssignmentProperties.java

```diff
@@ -15,6 +12,7 @@
 public final class KeyVaultRoleAssignmentProperties {
     private final String roleDefinitionId;
     private final String principalId;
+    private final KeyVaultRoleScope roleScope;
```

Comment:
Just `scope` - or at least the public "property", but I figured you'd want the field to match.

Reply:
Will rename to `scope`.